.gitignore

```diff
@@ -15,4 +15,7 @@ public/app.js
 
 # Keys
 apiKeys.js
-apikeys.js
+
+# Auth
+auth.js
```

Comment:
Maybe add the auth key to the apikeys.js file that contains the google and eventful api key?